mobile-app/cypress/e2e/functional/wallet/loans/loans.spec.ts

```diff
@@ -42,7 +42,9 @@ function checkTokensSortingOrder(
             .invoke("text")
             .then((text) => {
               const secondVal = text.split("% Interest")[0];
-              expect(Number(firstVal)).to.be.lessThan(Number(secondVal));
+              expect(Number.parseInt(firstVal)).to.be.lessThan(
+                Number.parseInt(secondVal)
```

Comment:
Hello, I happen to be working on loans e2e as well and saw similar issue here. This change will just parse numbers like `1,234` to `1`. 
I think what we need to address is the comma inside string before parsing into a number:
`Number("1,234") -> NaN`

Reply:
Hi @lykalabrada thanks for pointing out. But from my investigation, it does work to some extent. But let me know if you found much more accurate implementation. I also not so sure
![Screenshot 2022-10-26 151108](https://user-images.githubusercontent.com/11317652/197958842-23ffac9d-332f-49d2-8394-73f74208bd02.png)
![Screenshot 2022-10-26 151302](https://user-images.githubusercontent.com/11317652/197958992-f511217f-e217-48af-8e17-717636b4ce5b.png)

Reply:
Hmm yes this solution works for our current use case, but if in the future the data changed and we somehow got something like values `5 ≈ 5` and `1,000 ≈ 1` then this will fail.
I'm thinking, since having comma inside the value is the root cause of this NaN issue:
![image](https://user-images.githubusercontent.com/17063281/197980263-c427e7ad-f0ea-42c4-b027-94a7c1ccc7e4.png)

Then maybe we can remove the comma and parse to number:
```suggestion
              const value = text.split("$")[1];
              const secondVal = value.replace(/,/g, "");
              expect(Number(firstVal)).to.be.lessThan(Number(secondVal));
```

P.S. Issue is intermittent on current main branch. I don't know why but sometimes the value got commas and sometimes it doesn't have😖

Reply:
@lykalabrada tested yes that will work too. Anyway since you working on the loans e2e perhaps we could close this PR  and combine it with yours since it is a small adjustment on the files.